Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -168,6 +196,8 @@ public sealed class Server : ServerBase, IDisposable {
         }
 
         public override void DidChangeTextDocument(DidChangeTextDocumentParams @params) {
+            _analyzerCreationTcs.Task.Wait();
```

Comment:
Do we ever call this from UI thread?

Reply:
It is out-of-proc so technically there is no UI thread. `OutOfProcProjectAnalyzer' calls it in UpdateContent. In VS, however, this task completes in initialization so only VS Code may hit the wait.